packages/quantic/cypress/e2e/default-1/refine-modal-content/refine-modal-content.cypress.ts

```diff
@@ -89,4 +90,18 @@ describe('quantic-refine-content', () => {
       });
     });
   });
+
+  describe('when disableFacetOrdering property is set to true', () => {
+    it('should not render the facets inside the quantic-facets-manager', () => {
+      visitRefineContent({
+        disableFacetOrdering: true,
+      });
+
+      scope('when loading the page', () => {
+        Expect.displayFiltersTitle();
+        Expect.displayClearAllFiltersButton(false);
+        Expect.displayFacetManager(false);
+      });
```

Comment:
We should also make sure that the duplicated facets are displayed  properly as well in the case where the `disableFacetOrdering` property is set to true.